### ISSUE TITLE
File list in Publish to Connect dialog not keyboard accessible

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5951]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.x
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
Publish a folder of documents to Connect, to reach this dialog (has both standalone dialog and wizard variations depending on current state):

<img width="590" alt="2020-01-02_10-38-41" src="https://user-images.githubusercontent.com/10569626/71685938-93e24500-2d4e-11ea-88c1-efb525d7699b.png">

Try to navigate and manipulate the list of files with keyboard (and/or a screen reader).

### Describe the problem in detail
Cannot use this control via keyboard or screen reader.

### Describe the behavior you expected
Navigate and manipulate the control with keyboard and get appropriate screen reader feedback.

**Comment by gtritchie** [comment]
Actually, you can navigate between the checkboxes with Tab / Shift+Tab, and select them with spacebar, so it's not entirely inaccessible, just very unexpected (something that looks like a list should be navigable with arrow keys, and Tab should move to the next control, not the next item in the list).

**Comment by gtritchie** [comment]
Closing this. Although this might not be the preferred keyboard model, it is usable via keyboard and screen-reader and thus isn't worth messing with.
